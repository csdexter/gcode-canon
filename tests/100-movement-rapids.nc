(testing rapid movements)
M05 S600
M09
M23
M49
M69

M17

G15
G17
G21
G23
G28 X0.000 Y0.000 Z0.000
G40
G49
G50
G69
G90

G00 X0.000 Y0.000 Z0.000
G00 Y10.000
G00 X10.000
G00 Z10.000
G00 X0.000 Y0.000 Z0.000

M02