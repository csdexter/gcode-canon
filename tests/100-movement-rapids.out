MSG: WAR: Machine servos activated!
MSG: STA: Scanning input for programs (O words)
MSG: WAR: Machine servos activated!
MPOS,0.00,10.00,0.00
MPOS,10.00,10.00,0.00
MPOS,10.00,10.00,10.00
MPOS,0.00,0.00,0.00
MPOS,0.00,10.00,0.00
MPOS,10.00,10.00,0.00
MPOS,20.00,10.00,0.00
MPOS,20.00,10.00,10.00
MPOS,20.00,10.00,20.00
MPOS,20.00,10.00,30.00
MPOS,0.00,0.00,0.00
MPOS,0.00,254.00,0.00
MPOS,254.00,254.00,0.00
MPOS,254.00,254.00,254.00
MPOS,0.00,0.00,0.00
MPOS,0.00,254.00,0.00
MPOS,254.00,254.00,0.00
MPOS,508.00,254.00,0.00
MPOS,508.00,254.00,254.00
MPOS,508.00,254.00,508.00
MPOS,508.00,254.00,762.00
MPOS,0.00,0.00,0.00
MPOS,7.07,7.07,0.00
MPOS,7.07,17.07,0.00
